app/src/components/combobox/styles.tsx

```diff
@@ -13,7 +21,7 @@ export const comboBoxCSS = (theme: Theme) => css`
   .px-combobox-container {
     display: flex;
     flex-direction: row;
-    width: fit-content;
+    width: ${width != null ? width : "fit-content"};
```

Comment:
This is a super minor nit but I think mixing emotion with strings feels pretty messy. I think just managing the width via direct element style would be what I would choose so that we can eventually constrain this like `useStyles` in the component library (e.g. get type hints for good sizes etc.)

Reply:
sounds good, can update